debian/control

```diff
@@ -3,7 +3,7 @@ Section: libs
 Priority: extra
 Maintainer: Wido den Hollander <wido@widodh.nl>
 Build-Depends: debhelper (>= 9), openjdk-8-jdk | openjdk-7-jdk, genisoimage,
- python-mysql.connector, maven (>= 3) | maven3, python (>= 2.7)
+ python-mysql.connector, maven (>= 3) | maven3, python (>= 2.7), lsb-release
```

Comment:
@vincentbernat that should be okay, we can use Xenial to build pkgs for both Xenial and Precise. At some point we'll need to stop building packages for Precise.